### PR TITLE
Fixed result object and documentation for Matches in Search parameters

### LINKED ISSUE #1295: Example request with matches on search page is wrong
Correct me if I am wrong but, example request in https://docs.meilisearch.com/reference/features/search_parameters.html#matches is wrong. 

Correct query should be `shifu` and not `winter feast`. 

The fix should be in this file;
https://github.com/meilisearch/documentation/blob/master/.code-samples.meilisearch.yaml#L328

### PR DESCRIPTION
**Pull request by kennetvu**
# Pull Request

## What does this PR do?
Fixes #1295

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: reference/features/search_parameters.md [hunk 1]
```diff
@@ -368,17 +368,26 @@ You would get the following response with **information about the matches in the
 {
   "id": "50393",
   "title": "Kung Fu Panda Holiday",
-  "poster": "https://image.tmdb.org/t/p/w1280/gp18R42TbSUlw9VnXFqyecm52lq.jpg",
+  "poster": "https://image.tmdb.org/t/p/w500/rV77WxY35LuYLOuQvBeD1nyWMuI.jpg",
```
Comment (kennetvu) [inline reference/features/search_parameters.md:371]: Not sure if I should update this link.

File: reference/features/search_parameters.md [hunk 2]
```diff
@@ -368,17 +368,26 @@ You would get the following response with **information about the matches in the
...
   "overview": "The Winter Feast is Po's favorite holiday. Every year he and his father hang decorations, cook together, and serve noodle soup to the villagers. But this year Shifu informs Po that as Dragon Warrior, it is his duty to host the formal Winter Feast at the Jade Palace. Po is caught between his obligations as the Dragon Warrior and his family traditions: between Shifu and Mr. Ping.",
   "release_date": 1290729600,
+  "genres": ["Animation", "Family", "TV Movie"],
```
Comment (kennetvu) [inline reference/features/search_parameters.md:374]: Not sure if I should include this. It got added based on example file movies.json.

File: reference/features/search_parameters.md (hunk 1 above)
Comment (dichotommy) [reply reference/features/search_parameters.md:371]: Wow, wouldn't have even thought of doing that! Thanks 😊

File: reference/features/search_parameters.md (hunk 2 above)
Comment (dichotommy) [reply reference/features/search_parameters.md:374]: We would probably prefer to leave this off. Even though most of our documentation examples use movies as the index, they aren't intended to be accurate to movies.json. And we do anything we can to make code samples shorter... 🙈

Comment (kennetvu) [reply reference/features/search_parameters.md:374]: Okey, I will remove this change.

**Review by dichotommy — APPROVED** [review]
LGTM! Thanks again @kennetvu 😊 🎉

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**